x/auction/spec/06_begin_block.md

```diff
@@ -0,0 +1,20 @@
+<!--
+order: 6
+-->
+
+At the beginning of each block, auctions that have reached `EndTime` are closed. The logic to close auctions is as follows:
```

Comment:
Spec states auctions are closed at the beginning of each block, but `abci.go` defines `EndBlocker`.